Example/ViewController+FreeDrive.swift

```diff
@@ -116,7 +122,6 @@ extension ViewController {
             let branchNames = branchEdgeIdentifiers.flatMap { edgeNames(identifier: $0) }
             statusString += " at \(branchNames.joined(separator: ", "))"
         }
-        print(statusString)
```

Comment:
This print statement was intentional: otherwise, all the electronic horizon work above is for naught. Ideally there would be some UI associated with this output, but a print statement was more expedient at the time.

Reply:
Fixed in ca8be54d000308cc280c37a06079bd8b2e7e05a2.